### ISSUE TITLE
Ensure that token batch expiration does not exceed registration expiration

### ISSUE THREAD

**Issue by dlongley** [digitalbazaar/bedrock-tokenization#48]

**Comment by mandyvenables** [comment]
@dlongley Would the way to do this be to check if an expiration date exists on a document, and if so created a new property on the `entity` record that contains the expiration date, and then check that when creating token batches?

**Comment by dlongley** [comment]
That was a little hard to parse but I think I got the gist of what you were asking.

Something to keep in mind is that the expiration date on the registration document gets updated every time `_refresh()` is called -- and then we'd have to mirror that to any other place it was duplicated, which means more database writes. We'll need to come up with something sufficiently efficient here.

**Comment by mandyvenables** [comment]
@dlongley I was thinking this issue was for if the document itself had an expiration date, we would want to ensure the token batches were not issued for an expired document.

Is this issue for the expiration of the document itself, or the `expires` property in the `tokenization-registration` table, or should those be split into 2 different issues?

**Comment by dlongley** [comment]
Bumping this back to Priority 1 to ensure that expiration periods are sync'd to avoid incongruencies that could affect users.

**Comment by dlongley** [comment]
Fixed by #87.
